.idea/misc.xml

```diff
@@ -8,7 +8,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="11" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="zulu-17" project-jdk-type="JavaSDK">
```

Comment:
not really sure that this should be checked in, but the project now requires Java 17:
https://github.com/JetBrains/jvm-native-trusted-roots/blob/trunk/pom.xml#L107

Tests don't run in the IDE without changing this file

Reply:
Yes, it should be

Reply:
In trunk: https://github.com/JetBrains/jvm-native-trusted-roots/commit/c44be3a94a9c9c6dc281f1f2333eb5bcc5117f2d